modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Panel.js

```diff
@@ -26,18 +26,13 @@ const PanelItem = ({children}) => {
 };
 
 export default function Panel({
-	additionalProps: _additionalProps,
 	children,
 	collapsable,
 	collapseClassNames,
-	componentId: _componentId,
 	cssClass,
 	displayTitle,
 	displayType,
 	expanded,
-	locale: _locale,
-	portletId: _portletId,
```

Comment:
I'm going to add these back because it looks like removing them creates a weird behavior

![paneltag1](https://github.com/liferay-frontend/liferay-portal/assets/46778114/e7e00bfd-234c-4f04-a284-7736dff08907)